site/content/docs/user/rootless.md

```diff
@@ -9,41 +9,58 @@ menu:
 Starting with kind 0.11.0, [Rootless Docker](https://docs.docker.com/go/rootless/), [Rootless Podman](https://github.com/containers/podman/blob/master/docs/tutorials/rootless_tutorial.md) and [Rootless nerdctl](https://github.com/containerd/nerdctl/blob/main/docs/rootless.md) can be used as the node provider of kind.
 
 ## Provider requirements
+
 - Docker: 20.10 or later
 - Podman: 3.0 or later
 - nerdctl: 1.7 or later
 
 ## Host requirements
+
+### cgroups v2
+
 The host needs to be running with cgroup v2.
 Make sure that the result of the `docker info` command contains `Cgroup Version: 2`.
 If it prints `Cgroup Version: 1`, try adding `GRUB_CMDLINE_LINUX="systemd.unified_cgroup_hierarchy=1"` to `/etc/default/grub` and
 running `sudo update-grub` to enable cgroup v2.
 
-Also, depending on the host configuration, the following steps might be needed:
+Your host may also need to enable [cgroup delegation](https://systemd.io/CGROUP_DELEGATION/) for daemon-based controller runtimes.
+This is not required for daemonless runtimes, such as podman. Note that this procedure may
+[negatively impact performance](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/ZMKLS7SHMRJLJ57NZCYPBAQ3UOYULV65/).
 
-- Create `/etc/systemd/system/user@.service.d/delegate.conf` with the following content, and then run `sudo systemctl daemon-reload`:
+To enable cgroup delegation, perform the folowing actions:
 
-  ```ini
-  [Service]
-  Delegate=yes
-  ```
+1. As root, create the directory `/etc/systemd/system/user@.service.d/` if it does not already exist
+
+   ```sh
+   sudo mdkir -p /etc/systemd/system/user@.service.d/
+   ```
+2. As root, create the file `/etc/systemd/system/user@.service.d/delegate.conf` with the following content:
+
+   ```ini
+   [Service]
+   Delegate=yes
+   ```
+
+3. Reload the systemd daemon:
+
+   ```sh
+   sudo systemctl daemon-reload
+   ```
 
-  (This is not enabled by default because ["the runtime impact of
-  [delegating the "cpu" controller] is still too
-  high"](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/ZMKLS7SHMRJLJ57NZCYPBAQ3UOYULV65/).
-  Beware that changing this configuration may affect system
-  performance.)
+4. If using docker, reload the user docker daemon:
 
-  Please note that:
+   ```sh
+   systemctl --user restart docker
+   ```
 
-  - `/etc/systemd/system/user@.service.d/` directory needs to be created if not already present on your host
-  - If using Docker and it was already running when this step was done, a restart is needed for the changes to take
-    effect
-      {{< codeFromInline lang="bash" >}}
-      systemctl --user restart docker
-      {{< /codeFromInline >}}
+### Networking
 
-- Create `/etc/modules-load.d/iptables.conf` with the following content:
+Containers running in rootless mode are not typically loaded with host-level iptable modules.
+This breaks the behavior of most Ingress and Gateway controllers.
```

Comment:
Not specific to these controllers

Reply:
Better to phrase as "This breaks the behavior of many networking components, such as Ingress and Gateway controllers"?